matflow/param_classes/load.py

```diff
@@ -871,6 +871,70 @@ def uniaxial_cyclic(
             out.extend(cycle_i)
         return [cls(**i)._remember_name_args(None, i) for i in out]
 
+    @classmethod
+    def from_npz_file(
+        cls,
+        npz_file_path,
+        idx,
+    ) -> list[Self]:
+        """
+        Read loading data from a numpy .npz file and format it appropriately.
+        Designed for running large arrays of simulations which each use loadcase of the
+        corresponding index.
+
+        Parameters
+        ----------
+        npz_file_path: str
+            Filepath to the loadcase.npz file.
+            File contains dict of the following keys:
+            num_incs: 1D numpy array
+                Array of the total increments of each loadcase.
+                (total number of increments the damask simulation should undergo)
+            inc_size: 2D numpy array
+                Array of the amount of strain each loadstep of each loadcase should undergo in the damask simulation
+            inc_size_final: 2D numpy array
+                Array of amount of strain of final loadsteps
+            u_sampled_split: 4D numpy array
+                Array of strain matricies (3x3) for each loadstep (1st,2nd D) should undergo for each loadcase (3nd D),
+                for each simulation (4th D). Sampled from the elements of an FE model.
+            LE_eval_sampled_split: 3D numpy array
+                Array of principle component vectors of strain matricies (3x1), for each loadstep (1st D),
+                should undergo for each loadcase (2nd D), for each simulation (3rd D).
+            strain_rate: 1D numpy array of one float
+                Array of strain rate to be used for every simulation
```

Comment:
This looks like a scalar array from your example data, i.e. `np.array(0.01)`. To avoid confusion, I would not refer to it as an array:
```suggestion
                Scalar strain rate to be used for every simulation
```